ci/recipe/meta.yaml

```diff
@@ -13,6 +13,8 @@ build:
 
 requirements:
   host:
+    # TODO: Pin version of openssl that supports @rpath/libssl.1.1.dylib
+    - openssl =1.1.1q
```

Comment:
Is this still needed? Was it OS X where this happened?

Reply:
I don't remember honestly. I think we are good?

Reply:
https://github.com/qiime2/qiime2/commit/d7cc9becd534f11b43c88d47b622015b2ccc128d Looks like that was pinned for Mac, and things are working now, so I think we are good?